src/Draco.Compiler/Api/MetadataReference.cs

```diff
@@ -47,13 +53,22 @@ public static MetadataReference FromPeStream(Stream peStream)
         return new MetadataReaderReference(metadataReader);
     }
 
+    public MetadataReference DocumentationFromStream(Stream xmlStream)
```

Comment:
Since this is a builder method, let's follow the convention and call it something like `WithDocumentation`.